subsys/bluetooth/controller/hal/nrf5/rand.c

```diff
@@ -12,36 +12,60 @@
 #include "common/log.h"
 #include "hal/debug.h"
 
-#define RAND_RESERVED (4)
+enum rand_status {
```

Comment:
Use standard errors here, no need to introduce a new set of errors. Also return -ERR